src/service/real-time-config/real-time-config-impl.js

```diff
@@ -589,8 +624,8 @@ export class RealTimeConfigManager {
     this.modifyRtcConfigForConsentStateSettings();
     customMacros = this.assignMacros(customMacros);
     this.rtcStartTime_ = Date.now();
-    this.handleRtcForCustomUrls(customMacros, checkStillCurrent);
-    this.handleRtcForVendorUrls(customMacros, checkStillCurrent);
+    this.handleRtcForCustomUrls(customMacros, checkStillCurrent, element);
+    this.handleRtcForVendorUrls(customMacros, checkStillCurrent, element);
```

Comment:
nit: can we assume amp-script will never be used by vendor urls?

Reply:
I think we can remove this, if they are using a url they should just return the json.